apps/io-services-cms-webapp/src/utils/jira-proxy.ts

```diff
@@ -13,7 +13,7 @@ import {
 } from "../lib/clients/jira-client";
 
 const formatOptionalStringValue = (value?: string) =>
-  value || `{color:#FF991F}*[ DATO MANCANTE ]*{color}`;
+  value || `{color:#000000}*[ DATO MANCANTE ]*{color}`;
```

Comment:
If we don't need to have a specific color, we could remove it:

```suggestion
  value || `*[ DATO MANCANTE ]*`;
```